bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
Maybe it should be the old format string, but with (self.name, self.sec_index).  I think the confusion comes from there being both a self.name and a self.sec_list.  Right now, if someone has multiple locations that use sec_list = 'somatic', they don't know which one is being set.